packages/ui/src/components/chart.tsx

```diff
@@ -1,297 +0,0 @@
-
-export type ChartConfig = {
-  [k in string]: ({ color?: never; theme: Record<keyof typeof THEMES, string> } | { color?: string; theme?: never }) & {
-    icon?: React.ComponentType
-    label?: React.ReactNode
-  }
-}
-
-interface ChartContextProps {
-  config: ChartConfig
-}
-
-const ChartContext = React.createContext<ChartContextProps | null>(null)
-
-function ChartContainer({
-  children,
-  className,
-  config,
-  id,
-  ...props
-}: {
-  children: React.ComponentProps<typeof RechartsPrimitive.ResponsiveContainer>['children']
-  config: ChartConfig
-} & React.ComponentProps<'div'>) {
-  const uniqueId = React.useId()
-  const chartId = `chart-${id || uniqueId.replace(/:/g, '')}`
-
-  return (
-    <ChartContext.Provider value={{ config }}>
-      <div
-        className={cn(
-          "[&_.recharts-cartesian-axis-tick_text]:fill-muted-foreground [&_.recharts-cartesian-grid_line[stroke='#ccc']]:stroke-border/50 [&_.recharts-curve.recharts-tooltip-cursor]:stroke-border [&_.recharts-polar-grid_[stroke='#ccc']]:stroke-border [&_.recharts-radial-bar-background-sector]:fill-muted [&_.recharts-rectangle.recharts-tooltip-cursor]:fill-muted [&_.recharts-reference-line_[stroke='#ccc']]:stroke-border flex aspect-video justify-center text-xs [&_.recharts-dot[stroke='#fff']]:stroke-transparent [&_.recharts-layer]:outline-hidden [&_.recharts-sector]:outline-hidden [&_.recharts-sector[stroke='#fff']]:stroke-transparent [&_.recharts-surface]:outline-hidden",
-          className,
-        )}
-        data-chart={chartId}
-        data-slot="chart"
-        {...props}
-      >
-        <ChartStyle config={config} id={chartId} />
-        <RechartsPrimitive.ResponsiveContainer>{children}</RechartsPrimitive.ResponsiveContainer>
-      </div>
-    </ChartContext.Provider>
-  )
-}
-
-function useChart() {
-  const context = React.useContext(ChartContext)
-
-  if (!context) {
-    throw new Error('useChart must be used within a <ChartContainer />')
-  }
-
-  return context
-}
-
-const ChartStyle = ({ config, id }: { config: ChartConfig; id: string }) => {
-  const colorConfig = Object.entries(config).filter(([, config]) => config.theme || config.color)
-
-  if (!colorConfig.length) {
-    return null
-  }
-
-  return (
-    <style
-      dangerouslySetInnerHTML={{
-        __html: Object.entries(THEMES)
-          .map(
-            ([theme, prefix]) => `
-${prefix} [data-chart=${id}] {
-${colorConfig
-  .map(([key, itemConfig]) => {
-    const color = itemConfig.theme?.[theme as keyof typeof itemConfig.theme] || itemConfig.color
-    return color ? `  --color-${key}: ${color};` : null
-  })
-  .join('\n')}
-}
-`,
-          )
-          .join('\n'),
-      }}
-    />
-  )
-}
-
-const ChartTooltip = RechartsPrimitive.Tooltip
-
-function ChartTooltipContent({
-  active,
-  className,
-  color,
-  formatter,
-  hideIndicator = false,
-  hideLabel = false,
-  indicator = 'dot',
-  label,
-  labelClassName,
-  labelFormatter,
-  labelKey,
-  nameKey,
-  payload,
```

Comment:
Some of these properties are not typed and because we don't use the Chart component at the moment I decided to remove it. We can add it back later and figure out proper typing when we need it.